libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -135,6 +135,9 @@ class AC_WPNav
     /// reached_destination - true when we have come within RADIUS cm of the waypoint
     virtual bool reached_wp_destination() const { return _flags.reached_destination; }
 
+    /// set_reached_wp_destination - used to declare we have reached the target height during take-off
+    void set_reached_wp_destination() { _flags.reached_destination = true; }
```

Comment:
I think maybe we don't need this AC_WPNav::set_reached_wp_destination() anymore.